Libraries/LibCore/DateTime.cpp

```diff
@@ -36,6 +36,19 @@ DateTime DateTime::now()
     return from_timestamp(time(nullptr));
 }
 
+DateTime DateTime::create(unsigned year, unsigned month, unsigned day)
+{
+    DateTime dt = from_timestamp(time(nullptr));
+    dt.m_year = year;
+    dt.m_month = month;
+    dt.m_day = day;
+    dt.m_hour = 0;
+    dt.m_minute = 0;
+    dt.m_second = 0;
```

Comment:
The returned object will still have a mismatching `m_timestamp` :/

Reply:
Oh yeah that was kinda stupid, would something like this make sense? @awesomekling 

```
struct tm tm = { 0, 0, 0, (int)day, (int)month, (int)year, (int)dt.weekday(), (int)dt.day_of_year(), -1 };
dt.m_timestamp = mktime(&tm);
```

Reply:
Yes, I think so :)